exercises/practice/word-count/.docs/hints.md

```diff
@@ -0,0 +1,32 @@
+## General
+
+-  This exercise has many potential solutions and many paths you can take along the way. No path is manifestly "better" than another, although a particular path may be more interesting or better suited to what you want to learn or explore right now. Some paths may trade speed for clarity. Others might take up more memory but be more scalable or maintainable. We encourage you to try out more than one strategy to see what happens.
```

Comment:
This is pretty long, and it reads more like an introduction to the following bullets?  Should it be introductory text instead?  It does not quite read like the bullets that follow, either, which causes me to suspect it is something a little different.

Reply:
Since all of these are supposed to be bullets, I went the route of dividing it into three bullets of two sentences each.